src/zksync/bridge-utils/applyL1ToL2Alias.ts

```diff
@@ -0,0 +1,21 @@
+import type { Address } from 'abitype'
+import { pad, toHex } from '../../utils/index.js'
+import { l1ToL2AliasOffset } from '../constants/address.js'
+import { ADDRESS_MODULO } from '../constants/number.js'
+
+export type ApplyL1ToL2AliasParameters = {
+  address: Address
+}
+
+export type ApplyL1ToL2AliasReturnType = Address
+
+export function applyL1ToL2Alias(
```

Comment:
```suggestion

/** @internal */
export function applyL1ToL2Alias(
```

As per convo, I understand that all these utilities are not public API. Can we please mark them as internal?